packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -3,15 +3,69 @@ import type * as MessageTypes from '@trezor/blockchain-link-types/lib/messages';
 import { CustomError } from '@trezor/blockchain-link-types/lib/constants/errors';
 import { BaseWorker, ContextType, CONTEXT } from '../baseWorker';
 import { MESSAGES, RESPONSES } from '@trezor/blockchain-link-types/lib/constants';
-import { Connection, PublicKey } from '@solana/web3.js';
+import { Connection, ParsedTransactionWithMeta, PublicKey } from '@solana/web3.js';
```

Comment:
I haven't checked blockchain-link-types, but in case you are reexporting types from @solana/web3.js there, I would consider using that re-export. but no hard requirement at the moment